src/e2e-test/regression-test/util/handlers.js

```diff
@@ -0,0 +1,59 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+const _ = require('lodash');
+
+class Handler {
+    constructor (next) {
+        this.__next = next;
+    }
+
+    handle (data) {
+        if (!!this.__next && this.__next instanceof Handler) {
+            return this.__next.handle(data);
+        }
+        return data;
+    }
+}
+
+
+class DoNothingHandler extends Handler {
+    handle (data) {
+        return super.handle(data);
+    }
+}
+
+
+class ExtraCdaFieldHandler extends Handler {
```

Comment:
Do we test CDA and HL7 v2 data differently? It seems a CDA-specific handler is created here.

Reply:
Name has been modified for better understandability.